src/Arcus.Template.WebApi/Controllers/HealthController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using GuardNet;
+
+namespace Arcus.Template.WebApi.Controllers
+{
+    /// <summary>
+    /// API endpoint to check the health of the application.
+    /// </summary>
+    [ApiController]
+    [Route("api/v1/[controller]")]
```

Comment:
I'm not a fan of this, but can just be me :D

The reason is that if we rename the controller it will automatically break our API contract. Because of this I always explicitly add the route
```suggestion
    [Route("api/v1/health")]
```